App/packages/Newtonsoft.Json.10.0.3/LICENSE.md

```diff
@@ -0,0 +1,20 @@
+The MIT License (MIT)
```

Comment:
naive question - why do we commit the `packages/` folder in the first place?  Definitely a decision that happened prior to this PR, but adds a lot of noise here if we can get away with just references to nuget packages, and not inline the dependencies themselves.

Reply:
Back in the days of old, NuGet wasn't entirely reliable and NuGet proxy/cache feeds didn't exist, so in short: dependency reduction (not depending on say the internet to be up to build). Now, Stack has infrastructure or dependencies that make that a moot point so we can simplify :)